tests/keras/utils/vis_utils_test.py

```diff
@@ -25,6 +26,14 @@ def test_plot_model():
     vis_utils.plot_model(model, to_file='model2.png', show_shapes=True)
     os.remove('model2.png')
 
+    model = Sequential()
```

Comment:
Can you make it a separate function? `test_plot_sequential_embedding`. Also add the issue number as a docstring to say why this test is here.